main.js

```diff
@@ -28,13 +27,13 @@ let helpWindow;
 function createNewRuler(windowInfo) {
 	let _windowInfo = windowInfo || {};
 	let ruler = new BrowserWindow({
-		width: _windowInfo.width || 151,
-		height: _windowInfo.height || 126,
-		frame: false,
-		transparent: true,
-		alwaysOnTop: true,
-		x: _windowInfo.x,
-		y: _windowInfo.y,
+		'width': _windowInfo.width || 151,
+		'height': _windowInfo.height || 126,
+		'frame': false,
+		'transparent': true,
+		'alwaysOnTop': true,
+		'x': _windowInfo.x,
+		'y': _windowInfo.y,
```

Comment:
linters like it when property names are consistent in their quoting (either all quotes or no quotes, not a combination of both)